src/Microsoft.TestPlatform.CrossPlatEngine/EventHandlers/TestRequestHandler.cs

```diff
@@ -22,24 +22,25 @@ namespace Microsoft.VisualStudio.TestPlatform.CommunicationUtilities
 
     public class TestRequestHandler : ITestRequestHandler
     {
+        private const int MinimumProtocolVersionWithDebugSupport = 3;
```

Comment:
>MinimumProtocolVersionWithDebugSupport [](start = 26, length = 38)

Can be pulled in from a common place too.

Reply:
Duplicate, done.